checkov/cloudformation/graph_builder/graph_components/generic_resource_encryption.py

```diff
@@ -1,13 +1,15 @@
-from typing import Dict, List, Union, Any
+from __future__ import annotations
+
 from checkov.common.graph.graph_builder import EncryptionTypes
 from checkov.common.graph.graph_builder.graph_components.generic_resource_encryption_base import GenericResourceEncryptionBase
+from checkov.common.util.data_structures_utils import get_empty_list_str
 
 
 class GenericResourceEncryption(GenericResourceEncryptionBase):
     def __init__(
         self,
         resource_type: str,
-        attribute_values_map: Dict[str, Union[List[bool], List[str]]],
+        attribute_values_map: dict[str, list[bool] | list[str]],
```

Comment:
Can you teach me what's the difference here?
I know they both work, I'm just not sure why it should be important to change it to the second option

Reply:
as mentioned above it is the preferred way by `typeshed` https://github.com/python/typeshed/blob/main/CONTRIBUTING.md#conventions